src/utils/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const USER_ID = 11493;
+
+export const DOWNLOAD_ERROR = 'Unable to load todos';
+export const POST_ERROR = 'Unable to add a todo';
+export const TITLE_ERROR = 'Title should not be empty';
+export const DELETE_ERROR = 'Unable to delete a todo';
+export const UPDATE_ERROR = 'Unable to update a todo';
```

Comment:
You can create an object for your errors:

```javascript
export const ERRORS = {
  download: 'Unable to load todos',
  create: 'Unable to add a todo',
  invalidTitle: 'Title should not be empty',
  delete: 'Unable to delete a todo',
  update: 'Unable to update a todo',
} as const;
```
Something like this, but the naming could be better 😄